GetBPDCitizen/handler.ts

```diff
@@ -72,14 +72,24 @@ export const toApiBPDCitizen = (
           p =>
             ({
               ...citizen,
+              payment_instrument_enabled: citizen.payment_instrument_enabled,
               payment_instrument_hpan: p.hpan,
               payment_instrument_insert_date: citizen.payment_instrument_insert_date?.toISOString(),
+              payment_instrument_insert_user:
+                citizen.payment_instrument_insert_user,
               payment_instrument_status: p.status,
-              payment_instrument_update_date: citizen.payment_instrument_insert_date?.toISOString()
+              payment_instrument_update_date: citizen.payment_instrument_update_date?.toISOString(),
+              payment_instrument_update_user:
+                citizen.payment_instrument_update_user
```

Comment:
The values of `payment_instrument_enabled , payment_instrument_insert_user, payment_instrument_update_user` are already included in `...citizen` and can be removed

Reply:
sorry, removed